website/docs/cloud-docs/api-docs/team-tokens.mdx

```diff
@@ -62,6 +322,7 @@ This POST endpoint requires a JSON object with the following properties as a req
 | Key path                      | Type   | Default | Description                                                                                                     |
 | ----------------------------- | ------ | ------- | --------------------------------------------------------------------------------------------------------------- |
 | `data.type`                   | string |         | Must be `"authentication-token"`.                                                                               |
+| `data.attributes.description` | string | `null`  | The description of the team token, if supplied, otherwise the description will be `null`. The description **must** be unique within the context of the team, meaning legacy tokens may only have one `null` or one descriptive description at a time. |
```

Comment:
So this change includes adding a description field to the legacy team token API? Just confirming, as line 301 above implies that description is a new-API-only feature.